README.md

```diff
@@ -93,6 +93,8 @@ docker run --name marqo -it --privileged -p 8882:8882 --add-host host.docker.int
 
 ```
 
+Note: If your `marqo` container keeps getting killed, this is most likely due to too little memory being allotted to Docker. Increasing this limit (try 8GB) in your Docker settings may fix the problem.
```

Comment:
"this limit (try 8GB)", could this be "the memory limit for Docker to at least 6GB (8GB recommended)"